.github/workflows/ci.yml

```diff
@@ -6,8 +6,8 @@ on:
       - 'main'
 
 env:
-  BUILDER_VERSION: v0.9.73
-  BUILDER_SOURCE: releases
+  BUILDER_VERSION: xcodebuild_setup
```

Comment:
This would need to be updated once crt-builder is merged. Currently the crt-builder has CI failure related to gcc-8.